code/game/objects/items/flashlight.dm

```diff
@@ -37,6 +37,12 @@
 	update_action_button_icons()
 	update_icon()
 
+/obj/item/flashlight/attack_alien(mob/living/carbon/xenomorph/X, isrightclick = FALSE)
+	turn_light(X, FALSE)
```

Comment:
```suggestion
/obj/item/flashlight/attack_alien(mob/living/carbon/xenomorph/X, isrightclick = FALSE)
	if(!light_on)
		return ..()
	turn_light(X, FALSE)
```

Reply:
![image](https://user-images.githubusercontent.com/25491240/131219626-3b46f931-9bf3-4b78-89c1-97ea130db038.png)

Reply:
yeah but you'll get the attack animation and to_chat regardless